modules/static/opencensus/opencensus/initref

```diff
@@ -1 +1 @@
-d5d80953a8c2ff4633087af6933cd152678434bb
+v0.4.1
```

Comment:
This is the latest release: https://github.com/census-instrumentation/opencensus-proto/releases

The closest release to the original initref commit would have been:

```
➜  opencensus-proto git:(d5d8095) git describe --all d5d80953a8c2ff4633087af6933cd152678434bb
tags/v0.0.2-55-gd5d8095
```

This repo is archived and we don't expect any more updates.